ThunderRequest/TSCRequest.m

```diff
@@ -38,7 +45,7 @@ - (void)prepareForDispatch
 	}
 	
 	if (self.HTTPMethod == TSCRequestHTTPMethodGET && self.HTTPBody) {
-		NSLog(@"<ThunderRequest> Invalid request to: %@. Should not be sending a GET request with a non-nil body", self.URL.absoluteString);
+		os_log_error(request_log, "<ThunderRequest> Invalid request to: %@. Should not be sending a GET request with a non-nil body", self.URL.absoluteString);
```

Comment:
Are the prefixes of <ThunderRequest> necessary? What does this look like in console? Everything is broken into subsystems to avoid things like this AFAIK.

Reply:
No, you're correct. These can be removed now